### ISSUE TITLE
Bug: Custom Breakpoints Do Not Work On Col Elements

### ISSUE THREAD

**Issue by Jordan9232** [react-bootstrap/react-bootstrap#6370]
### Prerequisites

- [X] I am using the [correct version](https://github.com/react-bootstrap/react-bootstrap#bootstrap-compatibility) of React-Bootstrap for my version of Bootstrap
- [X] I have [searched](https://github.com/react-bootstrap/react-bootstrap/issues?q=is%3Aissue) for duplicate or closed issues
- [X] I have read the [contributing guidelines](https://github.com/react-bootstrap/react-bootstrap/blob/master/CONTRIBUTING.md)

### Describe the bug

- Custom breakpoints do not render properly on `Col` elements.
- Changing an existing breakpoint works fine and uses the correct width that I changed, this is only an issue with new breakpoints

### Expected behavior

- Using new breakpoints will render the breakpoint as a new prop like `<div class="col" xxs="4">` rather than `<div class="col-4">`
- I determined that this is not an issue with bootstrap itself, because when I manually changed the `class` attribute to what it should be in my browser's inspector it worked perfectly as it should.

### To Reproduce

- Adding a new `xxs` breakpoint and using `<Col xxs={6}>` will render as `<div class="col" xxs="6">`. It should be `<div class="col-6">`
- Adding a new `xxs` breakpoint and using `<Col xxs={6} xs={8}>` will render as `<div class="col-8" xxs="6">`. It should be `<div class="col-xs-8 col-6">`
- I also tried a new breakpoint somewhere between the existing breakpoints to see if it was just an issue with breakpoints smaller than any of the defaults and the same thing happens. Adding a new `xmd` breakpoint (between `md` and `lg`) and using `<Col md={4} xmd={5} lg={6}>` will render as `<div class="col-lg-6 col-md-4" xmd="5">`. It should be `<div class="col-lg-6 col-xmd-5 col-md-4">`

### Reproducible Example

- custom.scss:
```
@import 'bootstrap/scss/_functions.scss';
@import 'bootstrap/scss/_variables.scss';

$grid-breakpoints: (
  xxs: 0,
  xs: 392px,
  sm: 576px,
  md: 768px,
  xmd: 888px,
  lg: 992px,
  xl: 1200px,
  xxl: 1400px
);

$container-max-widths: (
  xs: 368px,
  sm: 540px,
  md: 720px,
  xmd: 852px,
  lg: 960px,
  xl: 1140px,
  xxl: 1320px
);

@import 'bootstrap/scss/bootstrap.scss';
```

- example.tsx:
```
import Col from "react-bootstrap/Col";
import Row from "react-bootstrap/Row";

export default function Example() {
  return (
    <Row>
      <Col xxs={12} xs={11} sm={10} md={9} xmd={8} lg={7} xl={6} xxl={5}>
        <p>xxs and xmd do not work properly here</p>
      </Col>
    </Row>
  );
}
```

### Screenshots

Inspector Elements:
![image](https://user-images.githubusercontent.com/36669088/172179174-da7efc74-6fdc-4bf8-9fc1-63c5e6e6f591.png)

Inspector Styles:
![image](https://user-images.githubusercontent.com/36669088/172179324-312a8084-6909-46c0-a978-07a189cf2a4c.png)

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome, Safari, Firefox, Microsoft Edge, Opera

### What version of React-Bootstrap are you using?

2.4.0

### What version of Bootstrap are you using?

5.1.3

**Comment by kyletsang** [comment]
Did you follow this?  https://react-bootstrap.github.io/layout/breakpoints/#custom-breakpoints

**Comment by Jordan9232** [comment]
Okay my bad, that did fix part of the problem at least. But there's still an issue with breakpoints smaller than the default `xs`. For example, normally if you do `<Col xs={12} sm={8}>` this will render as `<div class="col-12 col-sm-8">`, which is correct. 

If we're using the default breakpoints, since xs is the smallest it just uses the `col-12` class rather than `col-xs-12`. But now that's I've introduced a new breakpoint that is smaller than xs this causes problems. For example, now if I do `<Col xxs={12} xs={10}>` this becomes `<div class="col-10 col-xxs-12">` when it should be `<div class="col-12 col-xs-10">`.

Basically, I believe the Col component needs to check what the smallest breakpoint is rather than always assume it is xs. In fact, I believe I have spotted the exact line that is the culprit in the Col component:
```
const infix = brkPoint !== 'xs' ? `-${brkPoint}` : '';
```

I hope that makes sense.
